static/js/TextsPage.jsx

```diff
@@ -171,8 +171,7 @@ const Dedication = () => {
         dedicationData && (dedicationData.en || dedicationData.he) ?
         <div className="dedication">
           <span>
-              <span className="int-en" dangerouslySetInnerHTML={ {__html: dedicationData?.en} }></span>
-              <span className="int-he" dangerouslySetInnerHTML={ {__html: dedicationData?.he} }></span>
+              <InterfaceText markdown={{en: dedicationData?.en, he: dedicationData?.he}}/>
```

Comment:
@nissamai this turned out to be way easier than I expected. Also, much cleaner in the end. Again -- thanks.